site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/toolGroupMultipleDisplay.vm

```diff
@@ -30,7 +30,7 @@
                 #set ($escapedKey = $mapEntry.key.replaceAll(" ", "_"))
                     <li id="$escapedKey" class="">
                         <h4 class="$escapedKey specialLink">
-                            <a  #if ($groupCount ==1) class="open toggleNone"#else href="#" #end>
+                            <a  #if ($groupCount ==1) class="open toggleNone"#else #if ($!expandedGroups.contains($mapEntry.key)) class="open" #end href="#" #end>
```

Comment:
Can this just be an elseif ?

http://velocity.apache.org/engine/1.7/user-guide.html#if-elseif-else